osu.Game/Input/Bindings/DatabasedKeyBindingContainer.cs

```diff
@@ -21,7 +23,14 @@ public class DatabasedKeyBindingContainer<T> : KeyBindingContainer<T>
 
         private readonly int? variant;
 
-        private KeyBindingStore store;
+        private IDisposable realmSubscription;
+        private IQueryable<RealmKeyBinding> realmKeyBindings;
+
+        [Resolved]
+        private RealmKeyBindingStore store { get; set; }
```

Comment:
This isn't required anymore, right? Though it feels weird that everything's going through the `RealmContextFactory` - maybe it should expose a queryable instead of manually doing `realmFactory.Context.All<>()` to give it some purpose?

Reply:
Have removed this, doesn't seem used here.

I'm not sure we should add query methods just to give a class purpose. Last time we discussed the direction with realm the consensus was to do things as raw as possible, and we'll abstract out if/when that seems like a good option. Just to avoid the thread-thread passing as much as possible (make each usage responsible for its own detaching / whatever it decides to do).

Reply:
I agree with doing things raw, but it's hard to justify the "store" class existing when it's not really a store and instead a helper class of sorts.

Reply:
All usages of `RealmKeyBindingStore` are within `OsuGame` and `OsuGameBase`, so we can definitely move, rename or inline this. Open to any suggestions. For now maybe it can be renamed to `RealmKeyBindingRegistererererer` or something?